inc/Engine/Preload/ServiceProvider.php

```diff
@@ -59,10 +59,12 @@ public function register() {
 		$wp_file_system = $this->getContainer()->get( 'wp_direct_filesystem' );
 
 		$this->getContainer()->add( 'preload_caches_table', 'WP_Rocket\Engine\Preload\Database\Tables\Cache' );
-		$this->getContainer()->add( 'preload_caches_query', 'WP_Rocket\Engine\Preload\Database\Queries\Cache' );
+		$this->getContainer()->add( 'preload_caches_query', 'WP_Rocket\Engine\Preload\Database\Queries\Cache' )
```

Comment:
Since you are already here, I think it would be nice for us to update this ServiceProvider to use the name class resolution for all the registered container items like @Tabrisrp stated some weeks ago
```suggestion
		$this->getContainer()->add( 'preload_caches_query', Database\Queries\Cache::class )
```
what do you think?

Reply:
I got a PR stuck about that topic as @engahmeds3ed think it can have an impact we didn't saw with @Tabrisrp.  @engahmeds3ed at the end do we make the change?